### PR TITLE
Swope hcp - rewrite 

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

**Review by dockerymick — APPROVED** [review]
Your changes look good to me!

**Comment by dockerymick** [comment]
This PR has been opened for a couple of weeks and I don't see an issue related to it. I see it is approved, are we waiting on development to provide a review?

File: clusters/hosted_control_planes/hosted_sizing_guidance.adoc [hunk 1]
```diff
@@ -22,66 +22,66 @@ For bare-metal nodes, the default `maxPods` setting of 250 is likely to be a lim
...
+| Resource utilization with 1000 QPS
 | 9.0
 | 2.5
-|===
-
-By using these examples, you can factor in a load-based limit that is based on the expected rate of stress on the API, which is measured as the aggregated QPS across the 3 hosted API servers. For general sizing purposes, consider a 1000 QPS API rate to be a medium hosted cluster load and a 2000 QPS API to be a heavy hosted cluster load.
 
-To determine the API rate of an active hosted cluster, measure the number of queries per second for the appropriate hosted namespace by using the following metric, which you can query from the Administrator perspective in the {ocp-short} web console by selecting *Observe* -> *Metrics*:
+As the load increases by 1000 QPS, the hosted control plane resource utilization increases by 9 vCPUs and 2.5 GB memory. 
+//this content is left over--do we need
```
Comment (swopebe) [inline clusters/hosted_control_planes/hosted_sizing_guidance.adoc]: We need to check on this part.

**Review by rokej — APPROVED** [review]
Thank you @swopebe

**Pull request merged**